src/EnergyPlus/DElightManagerF.cc

```diff
@@ -565,7 +563,7 @@ namespace DElightManagerF {
 
                         // Is this RefPt hosted by current DElight Zone?
                         if (izone == refPt.ZoneNum) {
-                            auto &zn(state.dataHeatBal->Zone(izone));
+                            auto &thisZone = state.dataHeatBal->Zone(izone);
```

Comment:
[src/EnergyPlus/DElightManagerF.cc:568]:(style),[shadowVariable],Local variable 'zn' shadows outer variable

Reply:
Fixing shadows is always prone to causing diffs.  I would just always suggest making a commit before/after fixing shadow variables.   Maybe not each individual one, but at least in small chunks.  That way if diffs pop up, it's easier to go back and track down the problematic one.